test_opensearchpy/test_krbconnection.py

```diff
@@ -0,0 +1,30 @@
+""" Test module for the RequestsKrbConnections """
+
+import unittest
+from unittest.mock import patch
+
+from opensearchpy.connection.krb_requests import RequestsKrbConnection
+
+class TestKrbConnection(unittest.TestCase): # pylint: disable=invalid-name
+    """ Test for the RequestsKrbConnection """
+
+    @patch('kerberos.authGSSClientStep')
+    @patch('kerberos.authGSSClientResponse')
+    def test_krbConnection(self, authGSSClientResponse, authGSSClientStep): # pylint: disable=unused-argument,invalid-name
+        """ Check if we can we a Kerberos connection, mocking up the kerberos calls """
+
+        # Setting up the return value
+        authGSSClientResponse.return_value = "Mocking the response"
```

Comment:
Nit: make `return_value` similar to what's typically returned, e.g. `"Negotiate token"`.